Sources/UIKit+Diff.swift

```diff
@@ -15,21 +15,10 @@ public extension UITableView {
                       deleteRowAnimation: UITableViewRowAnimation = .automatic)
     {
         beginUpdates()
-        var insertionIndexPaths: [IndexPath] = []
-        var deletionIndexPaths: [IndexPath] = []
-        for step in diff {
-            switch step {
-            case .insert(let index):
-                insertionIndexPaths.append(IndexPath(row: index, section: section))
-            case .delete(let index):
-                deletionIndexPaths.append(IndexPath(row: index, section: section))
-            case .move(let from, let to):
-                moveRow(at: IndexPath(row: from, section: section),
-                                   to: IndexPath(row: to, section: section))
-            }
-        }
-        insertRows(at: insertionIndexPaths, with: insertRowAnimation)
-        deleteRows(at: deletionIndexPaths, with: deleteRowAnimation)
+        applyWithoutBatch(diff: diff,
```

Comment:
Reading it, I almost wonder if the name should be `applyWithinBatch`, to use the antonym of the meaning we did not originally mean.